Common/Data/Consolidators/TickConsolidator.cs

```diff
@@ -61,14 +63,24 @@ public TickConsolidator(Func<DateTime, CalendarInfo> func)
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TickConsolidator"/> class.
+        /// </summary>
+        /// <param name="period">The period.</param>
+        /// <param name="condition">The condition.</param>
+        public TickConsolidator(TimeSpan period, Func<Tick, bool> condition) : base(period)
+        {
+            _condition = condition ?? (t => true);
```

Comment:
Please add better documentation for the `period` and `condition` parameters.

Also, maybe renaming `_condition` to something like `_excludeCondition` would be more informative of what `_condition` is on first sight